docs/src/user_guide/time_series.md

```diff
@@ -350,7 +350,7 @@ Here's an example:
     add_time_series!(sys, component, ts)
 
     # Transform it to Deterministic
-    transform_single_time_series!(sys)
+    transform_single_time_series!(sys, DeterministicSingleTimeSeries, 24, Hour(24))
```

Comment:
@jd-lara note this change.  It will trigger a merge conflict in your docs-galore PR that you could easily miss.

Reply:
Amidst our churn you left the type in this function call.
```suggestion
    transform_single_time_series!(sys, 24, Hour(24))
```